tests/unit/backend/wmg/api/test_v1.py

```diff
@@ -441,137 +445,235 @@ def test__query_missing_tissue_request__returns_400(self):
     def test__query_request_with_filter_dims__returns_valid_filter_dims(
         self, load_snapshot, ontology_term_label, gene_term_label, fetch_datasets_metadata
     ):
+        # mock the functions in the ontology_labels module, so we can assert deterministic values in the
```

Comment:
I'm having trouble stepping through the code. Are you able to run this test under the debugger?

Reply:
discussed over slack